app/Models/Product.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+
+class Product extends Model
+{
+    use HasFactory;
+    protected $fillable = [
+        'name',
+        'description',
+        'price',
+        'category',
```

Comment:
Edit this to be category_id once you add the foreign key attribute on the migration table